.github/workflows/windows.yml

```diff
@@ -199,3 +199,117 @@ jobs:
           name: oqs-provider-msvc
           path: D:/a/oqs-provider/oqs-provider/_build/lib/oqsprovider.dll
 
+  msvc_native:
+# Run a job for each of the specified target architectures:
+    strategy:
+      matrix:
+        os:
+#          - windows-2019
+          - windows-2022
+        platform:
+          - arch: win64
+            oqsconfig: -DOQS_ALGS_ENABLED=STD
+            osslconfig: no-shared no-fips VC-WIN64A-masm
+        msarch:
+          - x64
+# - arm64
+# - x86
+    runs-on: ${{matrix.os}}
+    steps:
+        - name: Restore OpenSSL32 cache
+          id: cache-openssl32
+          uses: actions/cache@v3
+          with:
+            path: c:\openssl32
+            key: ${{ runner.os }}-msvcopenssl32
+        - uses: actions/checkout@v3
+        - name: Checkout OpenSSL master
+          if: steps.cache-openssl32.outputs.cache-hit != 'true'
+          uses: actions/checkout@v3
+          with:
+            set-safe-directory: true
+            repository: openssl/openssl
+            path: openssl
+        - uses: actions/checkout@v3
+          with:
+            set-safe-directory: true
+            repository: open-quantum-safe/liboqs
+            path: liboqs
+        - uses: ilammy/msvc-dev-cmd@v1
+          with:
+            arch: ${{ matrix.platform.arch }}
+        - name: Add msbuild to PATH
+          uses: microsoft/setup-msbuild@v1
+          with:
+            msbuild-architecture: ${{matrix.msarch}}
+            vs-version: '[17.4,]'
+            #Only windows 2022 image equiped with msvc of 17 version, build liboqs againt version 16 msvc will fail. 
+        - name: Setup nasm for OpenSSL build
+          uses: ilammy/setup-nasm@v1
+          if: steps.cache-openssl32.outputs.cache-hit != 'true'
+          with:
+            platform: ${{ matrix.platform.arch }}
+        - name: Setup perl for OpenSSL build
+          uses: shogo82148/actions-setup-perl@v1
+          if: steps.cache-openssl32.outputs.cache-hit != 'true'
+# OQS_USE_OPENSSL=OFF by default on Win32
+# if cmake --build fails, try explicit
+# cd _build && msbuild ALL_BUILD.vcxproj -p:Configuration=Release
+# fails:          cmake -DCMAKE_C_FLAGS="/wd5105" -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_PREFIX="c:\liboqs" ${{ matrix.platform.oqsconfig }} -S . -B _build
```

Comment:
Now, which of these commands can be used? It seems, nothing is executed in this branch (?)

Reply:
Or is this just a prior "build liboqs" trial (that probably should be removed)?

Reply:
deleted